### PR TITLE
Changed the algorithm to fetch the download filename

### LINKED ISSUE #1144: [sct_download_data] Impossible to download MNI_Poly_AMU template
The command `sct_download_data -d MNI-Poly-AMU` gives an error : 

capture d ecran 2017-01-31 15 11 40

### PR DESCRIPTION
**Pull request by peristeri**
Replace the regex with the parse_header function. Also removed the old
download function.

Fixes #1144

### CODE REVIEW

File: scripts/sct_download_data.py [hunk 1]
```diff
@@ -70,29 +70,29 @@ def main(args=None):
...
     url = dict_url[data_name]
+    printv('Downloading %s\n' % data_name, verbose)
     try:
-        # download_from_url(url, tmp_file)
         tmp_file = download_data(url, verbose)
     except (KeyboardInterrupt):
-        printv('\nERROR: User canceled process.', 1, 'error')
+        printv('\nERROR: User canceled process.\n', 1, 'error')
 
+    printv('Copy binaries to %s\n' % dest_folder, verbose)
```
Comment (SaraDupont) [inline scripts/sct_download_data.py]: Some of the data downloadable by `sct_download_data` are not binaries (for ex PAM50 and MNI-Poly-AMU are templates, sct_example_data and sct_testing_data are example/testing datasets, etc)

Maybe just print `Copy files to xx ` or  `Copy data to xx `

**Comment by jcohenadad** [comment]
@peristeri: in case of internet glitch the previews function could handle that-- is it still the case with your modification?

**Comment by peristeri** [comment]
@jcohenadad if you are refering to retries on failed attempts?

**Comment by jcohenadad** [comment]
yes, retry if connection pb

**Pull request merged**